pkg/webui/console/containers/device-payload-formatters/uplink.js

```diff
@@ -30,25 +35,39 @@ import sharedMessages from '@ttn-lw/lib/shared-messages'
 import PropTypes from '@ttn-lw/lib/prop-types'
 import attachPromise from '@ttn-lw/lib/store/actions/attach-promise'
 
+import { hexToBase64 } from '@console/lib/bytes'
+
 import { updateDevice } from '@console/store/actions/devices'
 
-import { selectSelectedApplicationId } from '@console/store/selectors/applications'
+import {
+  selectSelectedApplicationId,
+  selectApplicationLink,
+} from '@console/store/selectors/applications'
 import {
   selectSelectedDeviceId,
   selectSelectedDeviceFormatters,
+  selectSelectedDevice,
 } from '@console/store/selectors/devices'
 
+const m = defineMessages({
+  appFormatterText:
+    'Click <Link>here</Link> to modify the default uplink formatter for this application',
+})
```

Comment:
Let's use one message for both PF types. `Click <Link>here</Link> to modify the default payload formatter for this application`